### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by zubairov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,45 +1,91 @@
...
+`process`: - Current Node.js process,
+`require`: - Module require,
+`setTimeout`: - more on [setTimeout](https://nodejs.org/dist/latest-v5.x/docs/api/timers.html),
+`clearTimeout`: - more on [clearTimeout](https://nodejs.org/dist/latest-v5.x/docs/api/timers.html),
+`setInterval`: - more on setInterval,
+`clearInterval`: - more on clearInterval,
+`msg`: - Incoming message containing the payload from the previous step,
+`exports`: {},
+`messages`: - Utility for convenient message creation,
+`request`: - Http Client,
```
Comment (jhorbulyk) [inline README.md]: You should add a remark that this library is currently wrapped in Co.

File: README.md [hunk 2]
```diff
@@ -1,45 +1,91 @@
...
+### Doing complex data transformation
 
-```bash
-$ git clone https://github.com/elasticio/code-component.git code-component
+[JSONata](http://jsonata.org/) is great however sometimes it's easier to do things in JavaScript, if you want to transorm
+an incoming message with code, just use following sample:
 
-$ cd code-component
+```JavaScript
+async function run(msg) => {
```
Comment (jhorbulyk) [inline README.md]: Did you verify that this works?  My understanding is that the current version of this component does not support the `async` keyword.

File: README.md [hunk 3]
```diff
@@ -1,45 +1,91 @@
...
+      pass: process.env.ELASTICIO_API_KEY
+    },
+    json: true  
+  });
+  return {
+    fullName: res.body.first_name + " " + res.body.last_name,
+    email: res.body.email,
+    userID: res.body.id    
+  }
+}
```
Comment (jhorbulyk) [inline README.md]: Can you add some remarks that currently this component does not support snapshots or any form of credentials?

File: README.md (hunk 2 above)
Comment (zubairov) [reply README.md]: I did

**Comment by khanzadyan** [comment]
@zubairov and @jhorbulyk I suggest we merge this pull request.

**Comment by jhorbulyk** [comment]
@khanzadyan But it documents changes that as far as I can tell have not gone through the component QA process.

**Comment by khanzadyan** [comment]
I understand and accept your point @jhorbulyk , but as you remember we are under some deadline to deliver documentation. We can improve and change it in due course.

**Comment by jhorbulyk** [comment]
> we are under some deadline to deliver documentation

Documenting non-existant features is arguably not useful.

**Pull request merged**